src/pages/role_requests/RoleMembers.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react';
+import {useNavigate} from 'react-router-dom';
+
+import Alert from '@mui/material/Alert';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import CircularProgress from '@mui/material/CircularProgress';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import FormControl from '@mui/material/FormControl';
+import Grid from '@mui/material/Grid';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import Tooltip from '@mui/material/Tooltip';
+import Typography from '@mui/material/Typography';
+
+import {GridColDef} from '@mui/x-data-grid';
+
+import dayjs, {Dayjs} from 'dayjs';
+
+import {displayUserName} from '../../helpers';
+
+import {OktaUserGroupMember, RoleGroup} from '../../api/apiSchemas';
+import BulkRenewalDataGrid from '../../components/BulkRenewalDataGrid';
+
+interface Data {
```

Comment:
Bikeshed nit: `Data` is a pretty generic interface name. Probably could be named more relevant